pkg/controllers/management/drivers/kontainerdriver/kontainerdriver.go

```diff
@@ -65,43 +67,15 @@ func (l *Lifecycle) Create(obj *v3.KontainerDriver) (runtime.Object, error) {
 	v3.KontainerDriverConditionDownloaded.Unknown(obj)
 	v3.KontainerDriverConditionInstalled.Unknown(obj)
 
-	// Update status
-	obj, err := l.kontainerDrivers.Update(obj)
-	if err != nil {
-		return nil, err
-	}
-
-	if !obj.Spec.BuiltIn {
-		obj, err = l.download(obj)
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		v3.KontainerDriverConditionDownloaded.True(obj)
-		v3.KontainerDriverConditionInstalled.True(obj)
-	}
-
-	// Update status
-	obj, err = l.kontainerDrivers.Update(obj)
-	if err != nil {
-		return nil, err
-	}
-
-	if hasStaticSchema(obj) {
-		return obj, nil
-	}
-
-	err = l.createDynamicSchema(obj)
-	if err != nil {
-		return nil, err
-	}
-
-	v3.KontainerDriverConditionActive.True(obj)
-
```

Comment:
Removed all the codes? There aren't any operations in `Create` API?

Reply:
I would like the download process only happens in update function triggered by `Downloaded = Unknown` condition.